ketcher-autotests/tests/Macromolecule-editor/Macro-Micro-Switcher/macro-micro-switcher.spec.ts

```diff
@@ -0,0 +1,439 @@
+import {
+  turnOnMacromoleculesEditor,
+  turnOnMicromoleculesEditor,
+} from '@utils/macromolecules';
+import { Page, test } from '@playwright/test';
+import {
+  FILE_TEST_DATA,
+  TopPanelButton,
+  clickInTheMiddleOfTheScreen,
+  openFileAndAddToCanvas,
+  pressButton,
+  selectTopPanelButton,
+  takeEditorScreenshot,
+  takeMonomerLibraryScreenshot,
+  takePageScreenshot,
+  waitForLoad,
+  waitForPageInit,
+  waitForRender,
+} from '@utils';
+
+async function pasteFromClipboard(page: Page, fileFormats: string) {
+  await selectTopPanelButton(TopPanelButton.Open, page);
+  await page.getByText('Paste from clipboard').click();
+  await page.getByRole('dialog').getByRole('textbox').fill(fileFormats);
+  await waitForLoad(page, async () => {
+    await pressButton(page, 'Add to Canvas');
+  });
+}
+
+async function addToFavoritesMonomers(page: Page) {
+  await page.getByTestId('Bal___beta-Alanine').getByText('★').click();
+  await page
+    .getByTestId('Phe4Me___p-Methylphenylalanine')
+    .getByText('★')
+    .click();
+  await page.getByTestId('meM___N-Methyl-Methionine').getByText('★').click();
+  await page.getByTestId('RNA-TAB').click();
+  await page.getByTestId('summary-Sugars').click();
+  await page.getByTestId('25R___2,5-Ribose').getByText('★').click();
+  await page.getByTestId('summary-Bases').click();
+  await page.getByTestId('baA___N-benzyl-adenine').getByText('★').click();
+  await page.getByTestId('summary-Phosphates').click();
+  await page.getByTestId('bP___Boranophosphate').getByText('★').click();
+  await page.getByTestId('CHEM-TAB').click();
+  await page.getByTestId('Test-6-Ch___Test-6-AP-Chem').getByText('★').click();
+}
+
+test.describe('Macro-Micro-Switcher', () => {
+  test.beforeEach(async ({ page }) => {
+    await waitForPageInit(page);
+    await turnOnMacromoleculesEditor(page);
+  });
+
+  test('Check that preview window of macro structure doesn not change in micro mode ', async ({
+    page,
+  }) => {
+    /* 
+    Test case: https://github.com/epam/ketcher/issues/3603
+    Description: Preview window of macro structure doesn't change in micro mode
+    Test working incorrect now because we have bug https://github.com/epam/ketcher/issues/3603
```

Comment:
If this test works incorrectly due to some bug shouldn't there be added test.fail(); ?

Reply:
I think then it will always end up in failed and I don’t know how this will affect CI) The documentation says this: Playwright Test runs this test and ensures that it is actually failing.